src/core/utils.ts

```diff
@@ -33,6 +20,13 @@ export function trimLeft(str: string, char: string): string {
   return str.substring(leftEdge);
 }
 
+export function isPathInside(directory: string, filePath: string): boolean {
```

Comment:
comment? not 100% intuitive what this does

Reply:
Done